controller_manager:
  ros__parameters:
    update_rate: 1000


    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster


    joint_trajectory_controller:
      type: joint_trajectory_controller/JointTrajectoryController

joint_trajectory_controller:
  ros__parameters:
    joints:
      - panda_joint1
      - panda_joint2
      - panda_joint3
      - panda_joint4
      - panda_joint5
      - panda_joint6
      - panda_joint7
    interface_name: position
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity
